debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/J9ObjectFieldOffsetIterator_V1.java

```diff
@@ -222,8 +222,8 @@ private void fieldOffsetsFindNext() throws CorruptDataException {
 									if (valueTypeHelper.classRequires4BytePrePadding(fieldClass)) {
 										instanceSize = instanceSize.sub(U32.SIZEOF);
 									}
-									forceDoubleAlignment = modifiers.allBitsIn(J9JavaAccessFlags.J9AccVolatile)
-											&& instanceSize.eq(doubleSize);
+									forceDoubleAlignment = (modifiers.allBitsIn(J9JavaAccessFlags.J9AccVolatile) || valueTypeHelper.isRomClassAtomic(fieldClass.romClass()))
+															&& instanceSize.eq(doubleSize);
```

Comment:
Please don't increase indentation here (two tabs was and is sufficient).

Reply:
Fixed.